### PR TITLE
Updated `Create an exploration` page according to new UI changes.

### PR DESCRIPTION
**Pull request by sahiljoster32**
Here is the proof of changes being correct:
Screenshot from 2022-01-11 15-48-13

### CODE REVIEW

**Review by krishita30j — CHANGES_REQUESTED** [review]
@sahiljoster32 can you rename the png filename from libraryCreateButton to creatorDashboardCreateButton? Thanks.

**Comment by sahiljoster32** [comment]
@krishita30j Done!

**Comment by krishita30j** [comment]
@BenHenning PTAL! Thanks.

**Review by krishita30j — APPROVED** [review]

**Comment by krishita30j** [comment]
@seanlip PTAL!

File: pages/CreatingAnExploration.html [hunk 1]
```diff
@@ -9,10 +9,11 @@ <h2>
 
   <p>
     After you've played an exploration, why not try your hand at making one? To
-    do so, click the "Create" button in the top navbar.
+    do so, hover over your profile photo in the top right corner and click on 
+    “Creator Dashboard.” From there, click the “Create Exploration” button.
   </p>
 
-  <img src="static/images/libraryCreateButton.png" style="width: 100%;" alt="A menu bar at the top of the page, containing the 'Create' option circled in yellow, which is positioned on the right hand side of the menu bar.">
+  <img src="static/images/creatorDashboardCreateButton.png" style="width: 100%;" alt="A menu bar at the top of the page, containing the 'Create' option circled in yellow, which is positioned on the right hand side of the menu bar.">
```
Comment (seanlip) [inline pages/CreatingAnExploration.html]: The alt text seems wrong?

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks, just one comment.

Also one question -- when should this change be deployed to the live github.io site?

File: pages/CreatingAnExploration.html (hunk 1 above)
Comment (sahiljoster32) [reply pages/CreatingAnExploration.html]: Done!

**Comment by sahiljoster32** [comment]
@seanlip PTAL!

**Comment by seanlip** [comment]
I can't yet -- I still need an answer to the top-level question.

**Comment by sahiljoster32** [comment]
Looks like this site is deployed to the github pages. So, as soon as we update the main(oppia.github.io) repository, oppia.github.io site should updated accordingly. May be i'm wrong?
however, it would be better if update the site as soon as possible ( with next release), because this repository is not updated regularly!

**Comment by sahiljoster32** [comment]
@seanlip PTAL!

**Comment by seanlip** [comment]
OK, thanks -- I think that seems fine, we can update it immediately.

I just checked the Oppia site though, and the Create Exploration button positioning is weird. Should it look like this?
Screenshot from 2022-01-14 16-44-48

I would have thought that the button should be aligned with the top-right.

**Comment by sahiljoster32** [comment]
Since, some of the PR's are not deployed to oppia.org. 
as explaind by you in issue #14604
https://github.com/oppia/oppia/issues/14604
but now, in test server it is fixed.

https://user-images.githubusercontent.com/12553576/149486834-a8dd1074-5e83-4fde-9fff-0fe3eae19a91.mp4

**Comment by sahiljoster32** [comment]
@seanlip PTAL!

**Review by seanlip — APPROVED** [review]
OK, looks good, thanks!

**Pull request merged**